src/board/board.ts

```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk";
+import {drawChalkText, tileDataInterface} from "../tile/tile";
+
+type boardData = Array<Array<undefined|tileDataInterface>>
```

Comment:
```suggestion
type BoardData = Array<Array<TileData | undefined>>
```

good practice: put `undefined` last